genesis/process/disabled/disabled_test.go

```diff
@@ -271,12 +272,12 @@ func TestScheduledTxsExecutionHandler(t *testing.T) {
 	require.False(t, handler.IsInterfaceNil())
 
 	hash, err := handler.GetScheduledRootHashForHeader([]byte{})
-	require.Equal(t, make([]byte, 0), hash)
-	require.Nil(t, err)
```

Comment:
why this change? also on L280

Reply:
Because the stub was changed to return `nil, err` instead of `make(), nil`